markdown/ch5.markdown

```diff
@@ -1407,7 +1402,9 @@ But to be clear, we really have no single ground truth!
 from scipy import ndimage as ndi
 from skimage import color
 
-human_seg_url = 'http://www.eecs.berkeley.edu/Research/Projects/CS/vision/bsds/BSDS300/html/images/human/normal/outline/color/1122/108073.jpg'
+human_seg_url = ('http://www.eecs.berkeley.edu/Research/Projects/CS/'
```

Comment:
I understand your concern, but it's self-explanatory after spending a second to think about it, and most readers (I hope) would go on to try it in their open IPython console and learn something new! So I'd prefer to leave this as is. I used to type the + all the time until I realised it was redundant!